### PR TITLE
Added settings module with XMLParser class and Setting structure

### PR DESCRIPTION
**Pull request by AverinLV**

### CODE REVIEW

**Comment by AverinLV** [comment]
New paragraph in README! Yeah boyzz

File: VS/NeiroApp/NeiroApp/Settings/XMLParser.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  settings.push_back(s);
+  return true;
+}
+
+bool XMLParser::deleteSetting(unsigned long aId)
+{
+  for (auto it = settings.begin(); it != settings.end(); ++it)
+	{
+    if ((*it).id == aId) 
+		{
```
Comment (RazuvaevDD) [inline VS/NeiroApp/NeiroApp/Settings/XMLParser.cpp:92]: @AverinLV change it.

File: VS/NeiroApp/NeiroApp/Settings/tinyxml2.cpp [hunk 2]
```diff
@@ -0,0 +1,2837 @@
+/*
```
Comment (RazuvaevDD) [inline VS/NeiroApp/NeiroApp/Settings/tinyxml2.cpp:1]: @AverinLV rename this file to XMLlib.cpp (or LibXML or XMLParserLib or...)

Comment (RazuvaevDD) [reply VS/NeiroApp/NeiroApp/Settings/tinyxml2.cpp:1]: and hpp (h) file too.

File: VS/NeiroApp/NeiroApp/Settings/XMLParser.cpp [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+using namespace Settings_module;
+
+XMLParser::XMLParser(const char *aName):
+name{aName},
+settings{}
+{
+  std::fstream fs;
+  fs.open(aName);
+  if (fs.fail()) 
+	{
```
Comment (RazuvaevDD) [inline VS/NeiroApp/NeiroApp/Settings/XMLParser.cpp:13]: @AverinLV

**Comment by RazuvaevDD** [comment]
Check pls XMLParser.cpp.

**Pull request merged**